README.md

```diff
@@ -17,3 +17,10 @@ Supported text formats are:
 * HTML Table
 
 Please see the detailed documentation [here](text/README.md).
+
+* Excel
+
+## excel
```

Comment:
The excel heading is coming twice. Should be present once only.

Reply:
Addressed.